Remove duplicate 'svg/animations' tests in favor of WPT

Ahmad-S792 opened the pull request:
#### 2a92632912838a769ab5ec563052e34438298c03
<pre>
Remove duplicate &apos;svg/animations&apos; tests in favor of WPT

<a href="https://bugs.webkit.org/show_bug.cgi?id=266957">https://bugs.webkit.org/show_bug.cgi?id=266957</a>

Reviewed by Tim Nguyen.

In 272547@main, I did WPT sync from upstream and then compared the directories of
`svg/animations` and WPT directory and found these duplicates. This PR is to remove
local copies of tests in favor of upstream WPT and also update &apos;Test Expectations&apos; accordingly.

* LayoutTests/svg/animations/: Delete multiple tests
* LayoutTests/platform/glib/TestExpectations: Clean-up to remove deleted tests
* LayoutTests/platform/gtk/TestExpectations: Ditto
* LayoutTests/platform/ios-wk2/TestExpectations: Ditto
* LayoutTests/platform/ios/TestExpectations: Ditto
* LayoutTests/platform/mac-wk1/TestExpectations: Ditto
* LayoutTests/platform/mac-wk2/TestExpectations: Ditto
* LayoutTests/platform/mac/TestExpectations: Ditto
* LayoutTests/platform/wpe/TestExpectations: Ditto
* LayoutTests/TestExpectations: Ditto

Canonical link: <a href="https://commits.webkit.org/272558@main">https://commits.webkit.org/272558@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/6aadce3d7549eb25ec2a1e0c9d862ddfc335d20e

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/32091 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/48/builds/10809 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/14/builds/33879 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/34634 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/32/builds/29085 "Built successfully") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/49/builds/13166 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/51/builds/8032 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/28663 "Passed tests") | 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/32490 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/47/builds/9123 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/28680 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/7915 "Passed tests") | 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/42/builds/8088 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/10/builds/28595 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/35978 "Built successfully") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/29183 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/36/builds/29051 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/34196 "Passed tests") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/44/builds/8202 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/50/builds/6157 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/32054 "Passed tests") | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/46/builds/9843 "Built successfully") | | | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/7498 "Built successfully and passed tests") | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/43/builds/8848 "Built successfully") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/45/builds/8729 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

nikolaszimmermann commented on LayoutTests/svg/animations/svglength-additive-by-6.html:
Hmm, I remember that these test stem from the svg/dynamic-updates test harness.
You're saying this test has a 1:1 copy in WPT? Or only a test matching the name?

I'm curious to what level this needs the reviewers attention :)

Ahmad-S792 replied:
All of these tests were uploaded to WPT by Google Chrome in following bug: https://bugs.chromium.org/p/chromium/issues/detail?id=985335

Ahmad-S792 committed:
Remove duplicate 'svg/animations' tests in favor of WPT

https://bugs.webkit.org/show_bug.cgi?id=266957

Reviewed by Tim Nguyen.

In 272547@main, I did WPT sync from upstream and then compared the directories of
`svg/animations` and WPT directory and found these duplicates. This PR is to remove
local copies of tests in favor of upstream WPT and also update 'Test Expectations' accordingly.

* LayoutTests/svg/animations/: Delete multiple tests
* LayoutTests/platform/glib/TestExpectations: Clean-up to remove deleted tests
* LayoutTests/platform/gtk/TestExpectations: Ditto
* LayoutTests/platform/ios-wk2/TestExpectations: Ditto
* LayoutTests/platform/ios/TestExpectations: Ditto
* LayoutTests/platform/mac-wk1/TestExpectations: Ditto
* LayoutTests/platform/mac-wk2/TestExpectations: Ditto
* LayoutTests/platform/mac/TestExpectations: Ditto
* LayoutTests/platform/wpe/TestExpectations: Ditto
* LayoutTests/TestExpectations: Ditto

Canonical link: https://commits.webkit.org/272558@main
